vault/dendron.topic.schema.tutorial.first-schema.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 5U4eAiqshI67VxIL40KWH
+title: Making Your First Schema
+desc: >-
+  Learn to harness the powers of schema and automatically insert templates into
+  new notes
+updated: 1641257668806
+created: 1640784965050
+---
+
+## Summary
+
+You may find yourself wanting to use templates in your notes, especially when finding yourself using the same outline and headers over time. This tutorial will walkthrough making a template for daily journaling, using [The Five Minute Journal](https://www.intelligentchange.com/products/the-five-minute-journal) (5MJ) method, as an example, and automatically applying that template whenever you create a new journal.
```

Comment:
reword - can you use the following?

As you write more notes, you might find yourself re-using the same template across multiple notes. You can use schemas to automatically apply a template to notes at a given hierarchy. 

In this tutorial, we will walkthrough creating a schema for your [[daily journal|{link}]] that can automatically apply [The Five Minute Journal](https://www.intelligentchange.com/products/the-five-minute-journal) [[template|{link}]] to all your entries